core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.html

```diff
@@ -52,14 +54,12 @@
     </ng-template>
 
     <ng-template #emptyState>
-        <dot-pages-card-empty *ngFor="let item of [1, 2, 3, 4, 5]"></dot-pages-card-empty>
-
         <div class="dot-pages-empty__container">
```

Comment:
Are you deleting the files for the `dot-pags-card-empty` component?

Reply:
done!